src/tactic/interactive.lean

```diff
@@ -156,6 +156,7 @@ do h' ← get_unused_name `h,
 /--
 Similar to `refine` but generates equality proof obligations
 for every discrepancy between the goal and the type of the rule.
+`convert e using n` (n a nat) is also a thing (bounds the search).
```

Comment:
`using n` is similar to `congr' n`. You should mention that.

Reply:
done